### ISSUE TITLE
Returns true for posts which aren't valid

### ISSUE THREAD

**Issue by zeeshanmuhammad** [danhayden/dh-postcode#1]
This module doesn't seem to actually validate UK postcodes, there are additional checks missing, e.g:

*  The letters Q, V and X are not used in the first position
*  The letters I,J and Z are not used in the second position.
*  The only letters to appear in the third position are A, B, C, D, E, F, G, H, J, K, S, T, U and W.
*  The second half of the postcode is always consistent numeric, alpha, alpha format and the letters C, I, K, M, O and V are never used.

See the following resource for additional hints:
https://www.mrs.org.uk/pdf/postcodeformat.pdf

**Comment by SavageCore** [comment]
Indeed, the regex is far too basic. Is this project still alive? I'd say it should use [Google regex](https://i18napis.appspot.com/address/data/GB).
